ext/date/date_core.c

```diff
@@ -6732,13 +6732,13 @@ cmp_dd(VALUE self, VALUE other)
  *
  *     d <=> DateTime.new(2022, 7, 26) # => 1
  *     d <=> DateTime.new(2022, 7, 27) # => 0
- *     d <=> DateTime.new(2022, 7, 29) # => -1
+ *     d <=> DateTime.new(2022, 7, 28) # => -1
  *
  * - A numeric (compares <tt>self.ajd</tt> to +other+):
  *
- *     d <=> 2459789 # => -1
  *     d <=> 2459788 # => -1
  *     d <=> 2459787 # => 1
+ *     d <=> 2459786 # => -1
```

Comment:
I get `1` (rather than `-1`) returned for this example.

Reply:
Indeed.